plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -6708,47 +6715,162 @@ public void testAlterTableWithUnsupportedProperties()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testDropTableWithMissingMetadataFile()
```

Comment:
we have these tests on both the Iceberg BCT as well as on Iceberg BCST

why?

Reply:
BCT should have no less coverage than BCST

we need BCST so that all Catalog impls are exercised.